### ISSUE TITLE
Deep learning models in genomics; are we there yet?

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#4]
## TL;DR
This paper reviews deep learning architectures in genomics research (2020).

### Paper Link

https://www.sciencedirect.com/science/article/pii/S2001037020303068?via%3Dihub

### Author/Institution

Lefteris Koumakis (Foundation for Research and Technology - Hellas)

## Overview

This paper develops a novel deep learning approach that is able to successfully address current limitations of standard statistical approaches such as land marking and joint modeling. Dynamic-DeepHit flexibly incorporates the available longitudinal data comprising various repeated measurements (rather than only the last available measurements) in order to issue dynamically updated survival predictions for one or multiple competing risk(s).

## Methods

Deep learning, Genomics, Computational biology, Bioinformatics, Gene expression and regulation, Precision medicine

## Cite

Lefteris Koumakis, Deep learning models in genomics; are we there yet?, Computational and Structural Biotechnology Journal, Volume 18, 2020, Pages 1466-1473, ISSN 2001-0370, https://doi.org/10.1016/j.csbj.2020.06.017.

## Comments

Review paper
